ios/PolyPodApp/PolyPod/FilePicker.swift

```diff
@@ -11,7 +28,21 @@ class FilePicker: NSObject, UIDocumentPickerDelegate {
         return mimeToUti(nil)
     }
     
-    func pick(type: String?, completion: @escaping (URL?) -> Void) {
+    func loadExternalFileData(url: URL) -> ExternalFile {
+        var fileSize: Int64 = 0
+        do {
+            let fileAttribute: [FileAttributeKey : Any] =
+                try FileManager.default.attributesOfItem(atPath: url.path)
+            if let fileNumberSize: NSNumber = fileAttribute[FileAttributeKey.size] as? NSNumber {
+                fileSize = Int64(truncating: fileNumberSize)
+                }
```

Comment:
Nit: Odd indentation - same with the closing brace after `return` below. Selecting all text and hitting CTRL-i should fix it.